docs/source/etcd-data-model.rst

```diff
@@ -81,10 +81,18 @@ The object stored is a JSON blob with the following structure:
       "name": "<name of linux interface>",
       "mac": "<MAC of the interface>",
       "profile_ids": ["<profile_id>", …],
+      "ipv4_nat": [
+        {"in_ip": "10.0.0.1", "out_ip": "192.168.0.1"},
```

Comment:
Should the in_ip be 198.51.100.17 to be consistent in this example?  Or have I misunderstood how this is meant to work.  I was imagining it is mandatory that the in_ip falls within one of the ipv4_nets associated with this endpoint.  Ditto for ipv6 example.